### PR TITLE
Update libcamera documentation to include the libcamera-apps.

### PR DESCRIPTION
**Pull request by davidplowman**

### CODE REVIEW

**Comment by davidplowman** [comment]
@naushir @JamesH65 
I'm assuming James can merge this request? Let me know if I've done it wrong... thanks!

File: linux/software/libcamera/README.md [hunk 1]
```diff
@@ -9,71 +9,99 @@ sudo apt update
...
-We can now check out the code and configure the build for the Raspberry Pi as follows.
+Unfortunately, at the time of writing, the default version of meson is a little old, so please execute:
 
 ```bash
-git clone git://linuxtv.org/libcamera.git
-cd libcamera
-```
+wget http://ftp.br.debian.org/debian/pool/main/m/meson/meson_0.56.2-1_all.deb
+sudo dpkg -i meson_0.56.2-1_all.deb
+ ```
```
Comment (naushir) [inline linux/software/libcamera/README.md:50]: Not sure it matters, but I did a
`sudo pip3 install --update meson`

to get the latest version.

Comment (davidplowman) [reply linux/software/libcamera/README.md:50]: Oooh, that's much nicer, didn't know you could do that. I'll push that correction.

Comment (davidplowman) [reply linux/software/libcamera/README.md:50]: Though the magic seems to require "--upgrade", not "--update" !!

Comment (naushir) [reply linux/software/libcamera/README.md:49]: Oops, yes.  Wrote that (erroneously) from memory.

Comment (JamesH65) [reply linux/software/libcamera/README.md:49]: Age. Once you get to my age, you won't even remember your children's names! Anyway, must go, child 2 needs something...

**Comment by JamesH65** [comment]
LGTM, please indicate if you are happy for it to be merged as is.

**Comment by davidplowman** [comment]
Yes please, I'm happy!

**Pull request merged**